backend/src/main/java/fk/prof/backend/http/LeaderHttpVerticle.java

```diff
@@ -45,11 +46,21 @@ public LeaderHttpVerticle(ConfigManager configManager,
 
   @Override
   public void start(Future<Void> fut) {
+
+    Future serverInitFuture = Future.future();
+    Future associationInitFuture = Future.future();
```

Comment:
Initialize policy store and backend association store in leader elected task rather than the verticle.